docs/cloud/satellites/byoc/aws/manual.md

```diff
@@ -0,0 +1,172 @@
+
+### Instance Role
+
+Each satellite is configured to put relevant Buildkit logs in Cloudwatch. Earthly relies on an instance role to provide the relevant permissions.
+
+Start by creating a new IAM policy:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "logs:PutLogEvents",
+                "logs:CreateLogStream"
+            ],
+            "Resource": [
+                "arn:aws:logs:us-west-2:012345678901:log-group:/earthly/satellites/name:log-stream:*",
+                "arn:aws:logs:us-west-2:012345678901:log-group:/earthly/satellites/name"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+Follow [AWS' guide to create a new instance role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role), associating this new policy with your role. 
+
+### Compute Role
+
+Earthly uses a role within your AWS account to enable orchestration. This means that Earthly will only time-limited, user-revocable access to your cloud account.
+
+Start by creating the policy needed:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "tag:GetResources",
+                "iam:PassRole",
+                "ec2:DescribeSubnets",
+                "ec2:DescribeInstances",
+                "ec2:DescribeInstanceTypes",
+                "ec2:DescribeImages"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "ec2:RunInstances",
+                "ec2:ModifyInstanceAttribute"
+            ],
+            "Resource": [
+                "arn:aws:ec2:us-west-2::image/*",
+                "arn:aws:ec2:us-west-2:012345678901:volume/*",
+                "arn:aws:ec2:us-west-2:012345678901:security-group/sg-012345678901",
+                "arn:aws:ec2:us-west-2:012345678901:network-interface/*",
+                "arn:aws:ec2:us-west-2:012345678901:key-pair/name-satellite-key",
+                "arn:aws:ec2:us-west-2:012345678901:instance/*",
+                "arn:aws:ec2:us-west-2:012345678901:subnet/subnet-012345678901"
+            ],
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "ec2:TerminateInstances",
+                "ec2:StopInstances",
+                "ec2:StartInstances"
+            ],
+            "Resource": "arn:aws:ec2:us-west-2:012345678901:instance/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "ec2:DetachVolume",
+                "ec2:DeleteVolume",
+                "ec2:CreateTags",
+                "ec2:AttachVolume"
+            ],
+            "Resource": [
+                "arn:aws:ec2:us-west-2:012345678901:volume/*",
+                "arn:aws:ec2:us-west-2:012345678901:instance/*"
+            ],
+            "Effect": "Allow"
+        }
+    ]
+}
+```
```

Comment:
Another place where we need to update the policy if we change it?

Reply:
Perhaps we just say this is the basic policy needed and they can extend it if they want tighter security?

Reply:
Yep this is a 3rd place we will need to keep up to date with policy changes.